routers/api/v1/api.go

```diff
@@ -527,23 +535,6 @@ func RegisterRoutes(m *macaron.Macaron) {
 			})
 		})
```

Comment:
`}, reqToken())` needed somewhere?

Reply:
Done some extensive testing on all http://localhost:3000/api/v1/users/* API calls and all seems to be secure, in that everything public on GitHub is also the same in my branch of Gitea. I will keep testing this though, just to make sure all is secure.

Reply:
Oh, I see... the whole point was to allow annonymous access. 🤦‍♂

Reply:
Actually though your comment did catch something, that the tokens API was no longer usable with a token. I have fixed that.